packages/pigeon/example/app/macos/Runner/messages.g.h

```diff
@@ -56,11 +56,17 @@ NSObject<FlutterMessageCodec> *PGNExampleHostApiGetCodec(void);
 extern void SetUpPGNExampleHostApi(id<FlutterBinaryMessenger> binaryMessenger,
                                    NSObject<PGNExampleHostApi> *_Nullable api);
 
+extern void SetUpPGNExampleHostApiWithSuffix(id<FlutterBinaryMessenger> binaryMessenger,
+                                             NSObject<PGNExampleHostApi> *_Nullable api,
+                                             NSString *messageChannelSuffix);
+
 /// The codec used by PGNMessageFlutterApi.
 NSObject<FlutterMessageCodec> *PGNMessageFlutterApiGetCodec(void);
 
 @interface PGNMessageFlutterApi : NSObject
 - (instancetype)initWithBinaryMessenger:(id<FlutterBinaryMessenger>)binaryMessenger;
+- (instancetype)initWithBinaryMessenger:(id<FlutterBinaryMessenger>)binaryMessenger
+                   messageChannelSuffix:(NSString *)messageChannelSuffix;
```

Comment:
Optional: it would be a little more idiomatic to make this `nullable NSString *` and then convert `nil` to `@""` internally. (On the other hand, it's very unlikely anyone would ever use this method with no suffix, given the other method.)